source/code/providers/Container_DaemonEvent_Class_Provider.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Container_ContainerLogFileReader.h"
 
 #define LASTQUERYTIMEFILE "/var/opt/microsoft/docker-cimprov/state/LastEventQueryTime.txt"
+#define TEST_LASTQUERYTIMEFILE "/tmp/LastEventQueryTime.txt"

```

Comment:
I can do that, but I saw other places where /tmp/ was used as a folder location for the unit tests. I decided to follow their example.

________________________________
From: Jeff Coffler <notifications@github.com>
Sent: Tuesday, May 31, 2016 4:26:46 PM
To: Microsoft/Docker-Provider
Cc: Henry Rawas; Author
Subject: Re: [Microsoft/Docker-Provider] DaemonEvent unit test: use temp file location if testing, ignore non-… (#5)


In source/code/providers/Container_DaemonEvent_Class_Provider.cpp<https://github.com/Microsoft/Docker-Provider/pull/5#discussion_r65280188>:

> @@ -18,6 +18,7 @@
>  #include "Container_ContainerLogFileReader.h"
>
>  #define LASTQUERYTIMEFILE "/var/opt/microsoft/docker-cimprov/state/LastEventQueryTime.txt"
> +#define TEST_LASTQUERYTIMEFILE "/tmp/LastEventQueryTime.txt"


So this is a problem if multiple people work on the unit tests. Why not make this something like ./LastEventQueryTime.txt, which would allow two people (working on the same machine) to not clobber each other, since they'd be in different directory trees?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/Docker-Provider/pull/5/files/6ec8f97f1428cc3483fc2f7af571b1cf2ec59dfa#r65280188>, or mute the thread<https://github.com/notifications/unsubscribe/AAz3SPYrt96oOZGZGZ4eDVER2Q0HrEW6ks5qHMO2gaJpZM4Iq_GW>.